src/content/blog/2024-05-09-eslint-compatibility-utilities.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: Introducing ESLint Compatibility Utilities
+teaser: "Do you rely on a plugin that doesn't yet work with ESLint v9? This package will help."
+tags:
+  - Compatibility
+  - Tools
+authors:
+  - nzakas
+categories:
+  - Announcements
+---
+
+The release of ESLint v9.0.0 brought with it the rollout of the [new configuration system](/blog/2023/10/flat-config-rollout-plans/), but also a series of changes to the [rules API](/blog/2023/09/preparing-custom-rules-eslint-v9/). These changes are necessary in order to prepare ESLint for implementing [language plugins](https://github.com/eslint/rfcs/blob/main/designs/2022-languages/README.md), which will give ESLint the ability to natively lint languages other than JavaScript. As a result, plugin authors needed to update their rules to work with v9.0.0, and unfortunately, that means some of the plugins you rely on may not have been updated yet. That's why we're releasing the compatibility utilities.
+
+## How to know if the compatibility utilities will help
+
+These utilities may help if you encounter any of the following errors while running ESLint:
+
+```
+TypeError: context.getScope is not a function
+TypeError: context.getAncestors is not a function
+TypeError: context.markVariableAsUsed is not a function
+TypeError: context.getDeclaredVariables is not a function
+```
+
+These errors mean that the plugin rules have not been updated to latest ESLint rule API.
+
+## Using the compatibility utilities
+
+First, install the `@eslint/compat` package using npm or any npm-compatible CLI:
```

Comment:
```suggestion
First, install the [`@eslint/compat`](https://npmjs.com/package/@eslint/compat) package using npm or any npm-compatible CLI:
```

It would be good to have one link to the package on npm.